torch/csrc/jit/pybind.h

```diff
@@ -60,15 +60,17 @@ template <> struct type_caster<torch::jit::Symbol> {
 
   bool load(handle src, bool) {
     try {
-      value = torch::jit::Symbol(py::cast<std::string>(src));
+      // URK, isn't this suppressing errors from parseQualString?  What is going
+      // on here?
```

Comment:
It might be for the string cast (since `load` is generally not expected to throw). We should move the parsing part out of the try-catch